meson.build

```diff
@@ -43,7 +43,8 @@ warning_flags = [
   '-Waddress',
   '-Wno-multichar',
   '-Wvla',
-  '-Wpointer-arith'
+  '-Wpointer-arith',
+  '-Wno-sign-compare'
```

Comment:
plz try to fix build warning without the flag '-Wno-sign-compare'

Reply:
Updated PR as per your suggestion